pycm/pycm_handler.py

```diff
@@ -88,73 +31,24 @@ def __overall_stat_init__(cm):
-    cm.TPR_Macro = cm.overall_stat["TPR Macro"]
-    cm.FNR_Macro = cm.overall_stat["FNR Macro"]
-    cm.FPR_Macro = cm.overall_stat["FPR Macro"]
-    cm.PPV_Macro = cm.overall_stat["PPV Macro"]
-    cm.ACC_Macro = cm.overall_stat["ACC Macro"]
-    cm.TNR_Micro = cm.overall_stat["TNR Micro"]
-    cm.FPR_Micro = cm.overall_stat["FPR Micro"]
-    cm.TPR_Micro = cm.overall_stat["TPR Micro"]
-    cm.FNR_Micro = cm.overall_stat["FNR Micro"]
-    cm.PPV_Micro = cm.overall_stat["PPV Micro"]
-    cm.F1_Macro = cm.overall_stat["F1 Macro"]
-    cm.F1_Micro = cm.overall_stat["F1 Micro"]
-    cm.Overall_RACC = cm.overall_stat["Overall RACC"]
-    cm.Overall_RACCU = cm.overall_stat["Overall RACCU"]
-    cm.PI = cm.overall_stat["Scott PI"]
-    cm.AC1 = cm.overall_stat["Gwet AC1"]
-    cm.S = cm.overall_stat["Bennett S"]
-    cm.Kappa_SE = cm.overall_stat["Kappa Standard Error"]
-    cm.Kappa_CI = cm.overall_stat["Kappa 95% CI"]
-    cm.Chi_Squared = cm.overall_stat["Chi-Squared"]
-    cm.Phi_Squared = cm.overall_stat["Phi-Squared"]
-    cm.KappaUnbiased = cm.overall_stat["Kappa Unbiased"]
-    cm.KappaNoPrevalence = cm.overall_stat["Kappa No Prevalence"]
-    cm.V = cm.overall_stat["Cramer V"]
-    cm.DF = cm.overall_stat["Chi-Squared DF"]
-    cm.CI95 = cm.overall_stat["95% CI"]
-    cm.SE = cm.overall_stat["Standard Error"]
-    cm.ReferenceEntropy = cm.overall_stat["Reference Entropy"]
-    cm.ResponseEntropy = cm.overall_stat["Response Entropy"]
-    cm.CrossEntropy = cm.overall_stat["Cross Entropy"]
-    cm.JointEntropy = cm.overall_stat["Joint Entropy"]
-    cm.ConditionalEntropy = cm.overall_stat["Conditional Entropy"]
-    cm.MutualInformation = cm.overall_stat["Mutual Information"]
-    cm.KL = cm.overall_stat["KL Divergence"]
-    cm.LambdaB = cm.overall_stat["Lambda B"]
-    cm.LambdaA = cm.overall_stat["Lambda A"]
-    cm.HammingLoss = cm.overall_stat["Hamming Loss"]
-    cm.ZeroOneLoss = cm.overall_stat["Zero-one Loss"]
-    cm.NIR = cm.overall_stat["NIR"]
-    cm.PValue = cm.overall_stat["P-Value"]
-    cm.Overall_CEN = cm.overall_stat["Overall CEN"]
-    cm.Overall_MCEN = cm.overall_stat["Overall MCEN"]
-    cm.Overall_MCC = cm.overall_stat["Overall MCC"]
-    cm.RR = cm.overall_stat["RR"]
-    cm.CBA = cm.overall_stat["CBA"]
-    cm.AUNU = cm.overall_stat["AUNU"]
-    cm.AUNP = cm.overall_stat["AUNP"]
-    cm.RCI = cm.overall_stat["RCI"]
-    cm.C = cm.overall_stat["Pearson C"]
-    cm.SOA5 = cm.overall_stat["SOA5(Cramer)"]
-    cm.SOA6 = cm.overall_stat["SOA6(Matthews)"]
-    cm.CSI = cm.overall_stat["CSI"]
-    cm.ARI = cm.overall_stat["ARI"]
-    cm.B = cm.overall_stat["Bangdiwala B"]
-    cm.Alpha = cm.overall_stat["Krippendorff Alpha"]
-    cm.SOA7 = cm.overall_stat["SOA7(Lambda A)"]
-    cm.SOA8 = cm.overall_stat["SOA8(Lambda B)"]
-    cm.SOA9 = cm.overall_stat["SOA9(Krippendorff Alpha)"]
-    cm.SOA10 = cm.overall_stat["SOA10(Pearson C)"]
+    for stat, field_name in OVERALL_PARAMS2FIELD_NAME.items():
```

Comment:
I have a suggestion for this part:

1. We already have a `CLASS_PARAMS` in `pycm_params.py`, convert it to a dictionary that map `name` to `value` (they are the same at this moment)
2. Rename  `OVERALL_PARAMS2FIELD_NAME` to `OVERALL_PARAMS`
3. We need to update wherever these variables are used
4. We need to update `CONTRIBUTING.md`